events/wmt11.md

```diff
@@ -1,6 +1,164 @@
 ---
 parent: Events
 title: WMT11
-layout: coming_soon
-description: Conference on Machine Translation
+description: Workshop on Machine Translation
+location: Edinburgh, UK
+name: WMT11
+startDate: 2011-07-30
+
+seo:
+  type: Event
+  name: WMT11
+  description: Workshop on Machine Translation
+  startDate: 2011-07-30
+  endDate: 2011-07-31
+  eventAttendanceMode: OfflineEventAttendanceMode
+  eventStatus: EventScheduled
+
+  location:
+    type: PostalAddress
+    addressCountry: UK
+    addressLocality: Edinburgh
+
+  organizer:
+    type: Person
+    url: https://www.statmt.org/wmt11/
 ---
+
+The **Sixth Workshop on Machine Translation** (**WMT11**) took place from 30 July to 31 July, 2011, in Edinburgh, UK.
+
+[statmt.org/wmt11/program.html](https://www.statmt.org/wmt11/program.html)
+
+{% include collapsible_toc.html %}
+
+## Tasks
+
+- [Translation](https://statmt.org/wmt11/translation-task.html)
+- [Featured translation](https://statmt.org/wmt11/featured-translation-task.html)
+- [System combination](https://statmt.org/wmt11/system-combination-task.html)
+- [Evaluation](https://statmt.org/wmt11/evaluation-task.html)
+- [Tunable metrics](https://www.statmt.org/wmt11/tunable-metrics-task.html)
+
+### Tunable metrics task
+
+The tunable metrics task is an invitation-only shared task at WMT11. 
+
+## Schedule
+
+### Day 1
+
+Saturday, July 30, 2011
+
+|     |     |
+| --- | --- |
+| 9:00 – 9:25 | **Session 1: Shared Tasks and Evaluation** <br>(Chair: Chris Callison-Burch) <br>[***A Grain of Salt for the WMT Manual Evaluation***](https://statmt.org/wmt11/pdf/WMT01.pdf) <br>Ondrej Bojar, Miloš Ercegovčević, Martin Popel and Omar Zaidan |
```

Comment:
For paper links, it's better to link to the ACL Anthology or ArXiv websites so that we ensure the paper is available if statmt is ever down.

In this case: [aclanthology.org/W11-2101.pdf](https://aclanthology.org)/W11-2101.pdf